packages/ui/src/domainModules/collectionMammals/keyObjectModule.js

```diff
@@ -10,11 +10,13 @@ const {
   actionPrefix: 'MAMMALS',
   initialValues: {
     filterColumnIsOpen: false,
+    focusedSpecimenId: 1,
```

Comment:
this cannot have an initial value, because there is no default specimen :-) so remove this